ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
It sounds like unnecessary complication.
Why should I spend time on this, if batch script works good?
What **exactly** is wrong in os-dependend file?